configs/KAKUTEF7V2/config.h

```diff
@@ -80,31 +82,29 @@
 #define USB_DETECT_PIN       PA8
 
 #define TIMER_PIN_MAPPING \
-    TIMER_PIN_MAP( 0, PE13, 1,  1) \
-    TIMER_PIN_MAP( 1, PB0 , 2,  0) \
-    TIMER_PIN_MAP( 2, PB1 , 2,  0) \
-    TIMER_PIN_MAP( 3, PE9 , 1,  2) \
-    TIMER_PIN_MAP( 4, PE11, 1,  1) \
-    TIMER_PIN_MAP( 5, PC9 , 2,  0) \
-    TIMER_PIN_MAP( 6, PA3 , 2,  0) \
-    TIMER_PIN_MAP( 7, PD12, 1,  0)
-
+    TIMER_PIN_MAP( 0, RX_PPM_PIN        , 1,  1) \
+    TIMER_PIN_MAP( 1, MOTOR1_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 2, MOTOR2_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 3, MOTOR3_PIN        , 1,  2) \
+    TIMER_PIN_MAP( 4, MOTOR4_PIN        , 1,  1) \
+    TIMER_PIN_MAP( 5, MOTOR5_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 6, MOTOR6_PIN        , 2,  0) \
+    TIMER_PIN_MAP( 7, LED_STRIP_PIN     , 1,  0) \
+    TIMER_PIN_MAP( 8, CAMERA_CONTROL_PIN, 1,  0)
```

Comment:
E13 is assigned to both RX_PPM and CAMERA_CONTROL

Reply:
Remove RX_PPM_PIN :+1: